### PR TITLE
init

### PR DESCRIPTION
**Pull request by Artificial-Friend**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-core</artifactId>
+            <version>5.4.27.Final</version>
+        </dependency>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-java8</artifactId>
+            <version>5.4.27.Final</version>
+        </dependency>
```
Comment (Max-Pochepets) [inline pom.xml]: this library is not used here

File: src/main/java/foxtrot/uniform/charlie/kilo/dao/implementations/MovieDaoImpl.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            throw new DataProcessingException("ERROR: can't add movie " + movie, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+    }
+
+    @Override
+    public List<Movie> getAll() {
```
Comment (Max-Pochepets) [inline src/main/java/foxtrot/uniform/charlie/kilo/dao/implementations/MovieDaoImpl.java:38]: does this method signature match to corresponding interface?

File: src/main/java/foxtrot/uniform/charlie/kilo/Main.java [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+public class Main {
+    private static final Injector injector = Injector.getInstance("foxtrot.uniform.charlie.kilo");
+
+    public static void main(String[] args) {
+        Movie movie = new Movie();
+        movie.setTitle("Fast and Furious");
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
+        movieService.add(movie);
+
+        movieService.getAll().forEach(System.out::println);
```
Comment (Max-Pochepets) [inline src/main/java/foxtrot/uniform/charlie/kilo/Main.java:16]: if your service throws Exception, shouldn't we handle it?

**Review by Max-Pochepets — CHANGES_REQUESTED** [review]
make sure all your files finish with an empty line

**Review by Max-Pochepets — APPROVED** [review]
👍

File: .gitignore [hunk 4]
```diff
@@ -0,0 +1,3 @@
+.idea/*
+target/*
+*.iml
```
Comment (Sofasmile) [inline .gitignore]: let's add an empty line at the end of the file

File: .travis.yml [hunk 5]
```diff
@@ -0,0 +1,9 @@
+language: java
+jdk:
+  - openjdk11
+script:
+  - mvn clean install
+cache:
+  directories:
+    - $HOME/.m2
+    -
```
Comment (Sofasmile) [inline .travis.yml]: let's add an empty line at the end of the file

**Review by Sofasmile — CHANGES_REQUESTED** [review]

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**